utility/utils.py

```diff
@@ -533,20 +534,23 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
-def email_results(results_list, run_id, trigger_user, run_dir, suite_run_time, send_to_cephci=False):
+def email_results(email_result):
```

Comment:
IMO, `email_result` should be `test_result` or `test_data` as it contains information about the tests/suites...